### ISSUE TITLE
Class Annotation Properties

### ISSUE THREAD

**Issue by mark-jensen** [SDG-InterfaceOntology/sdgio#28]
Following from https://github.com/SDG-InterfaceOntology/sdgio/issues/26
We should come to agreement on a minimal set of desired annotations for SDGIO classes. 

In addition to ‘label’ and ‘definition’, we should aim to include

definition source
term editor
has curation status

Alternative terms and examples of usage can also be helpful sometimes.

The use of 'editor note' and 'comment' are not consistent in practice, so I recommend we establish some protocol of when to use one or the other, or if not to use one.

**Comment by pbuttigieg** [comment]
> definition source

Previously, I've added "data_base_cross_reference" on the definition Annotation. 
I don't mind using definition source - is there an agreed syntax for refs?

> term editor

I suggest we use ORCIDs

>Alternative terms and examples of usage can also be helpful sometimes.

I'd prefer synonyms here, is there a rationale to use alternative terms instead / along with synonyms?

>The use of 'editor note' and 'comment' are not consistent in practice, so I recommend we establish some protocol of when to use one or the other, or if not to use one. 

Agreed. I'd say editor notes should be more about the inner workings of the ontology (perhaps not to be exposed on the public landing pages), while comments are intended to guide users on class usage.

**Comment by cmungall** [comment]
I recommend follow the same pattern as ENVO and using axiom annotations

Term editor: lets add this automatically using the id-ranges. Note we need an id ranges file, [here is the envo one](https://github.com/EnvironmentOntology/envo/blob/master/src/envo/envo-idranges.owl)

> Agreed. I'd say editor notes should be more about the inner workings of the ontology (perhaps not to be exposed on the public landing pages), while comments are intended to guide users on class usage.

+1

Also useful:

 * http://purl.obolibrary.org/obo/IAO_0000232 curator note
 * http://purl.obolibrary.org/obo/IAO_0000112 example

**Comment by mark-jensen** [comment]
>I'd prefer synonyms here, is there a rationale to use alternative terms instead / along with synonyms?

I believe 'alternative term' denotes an exact match, an abbreviation or alternate spelling: 'MS' is an alternative term for 'multiple sclerosis'. Use of 'synonym' is less precise. However, in practice I haven't seen much in the way of consistent use. I am fine with use of either for now.

**Comment by pbuttigieg** [comment]
I've added  a draft sdgio-idranges.owl file based on the one ENVO uses in https://github.com/SDG-InterfaceOntology/sdgio/commit/12bb98e97f077ffea728f17e5a62c23e5379ca88. 

I assigned 00000000 to 00010000 to myself.

It's PURL is noted in the "Ontology:" field and I listed it as "https://purl.unep.org/sdg/sdgio-idranges.owl". This doesn't resolve yet, but then, neither does "http://purl.obolibrary.org/obo/envo/envo-idranges.owl".
